File: 2.0/js/webhooks/createWebhook.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.webhooks.createWebhook({
      body: {
        name: 'sms sent',
        resource: 'OutboundText',
        events: ['Finished'],
        callback: 'https://callback-service.com/listener'
      }
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch((err) => {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/webhooks/createWebhook.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.webhooks.createWebhook(
    body={
        'name': 'sms sent',
        'resource': 'OutboundText',
        'events': ['Finished'],
        'callback': 'https://callback-service.com/listener'
    }
).result()

print(response)
